src/parse/matches/arg_matches.rs

```diff
@@ -883,7 +883,7 @@ impl ArgMatches {
     }
 
     /// Because [`Subcommand`]s are essentially "sub-[`App`]s" they have their own [`ArgMatches`]
-    /// as well.But simply getting the sub-[`ArgMatches`] doesn't help much if we don't also know
+    /// as well. But simply getting the sub-[`ArgMatches`] doesn't help much if we don't also know
```

Comment:
nit: I feel like either the `.` or `but` should be removed.

Reply:
Merge as-is would be fine, didn't really plan on fixing anything else.